README.md

```diff
@@ -52,9 +52,9 @@ In order to submit your app to the App Store, you will need to eventually add th
 
 ### Configure Android
 
-- Add `apply from: '../node_modules/@unimodules/core/settings.gradle'` and then `useUnimodules.apply()` to the top of  `android/settings.gradle`
-  - If you need to customize the path to node_modules, for example because you are using yarn workspaces, then you can pass in a param for this: `useUnimodules.apply(nodeModulesPath: "../../../node_modules")`
-- Add `apply from: '../../node_modules/@unimodules/core/build.gradle'` anywhere in `android/app/build.gradle`
+- Add `apply from: '../node_modules/react-native-unimodules/gradle.groovy'` and then `includeUnimodules()` to the top of  `android/settings.gradle`
+  - If you need to customize the path to node_modules, for example because you are using yarn workspaces, then you can pass in a param for this: `includeUnimodules([modulesPaths: ['../../../node_modules']])`
+- Add `apply from: '../../node_modules/react-native-unimodules/build.gradle'` anywhere in `android/app/build.gradle`
```

Comment:
```suggestion
- Add `apply from: '../../node_modules/react-native-unimodules/gradle.groovy'` anywhere in `android/app/build.gradle` and then `useUnimodules()` inside `dependencies {}` block.
- If you need to customize the path to node_modules, for example because you are using yarn workspaces, then you can pass in a param `modulesPaths` for both of these functions: `includeUnimodules([modulesPaths: ['./path/to/node_modules']])`
```

Reply:
```suggestion
- Add `apply from: '../../node_modules/react-native-unimodules/gradle.groovy'` anywhere in `android/app/build.gradle` and then `useUnimodules()` inside `dependencies {}` block.
- If you need to customize the path to node_modules, for example because you are using yarn workspaces, then you can pass in a param `modulesPaths` for both of these functions: `includeUnimodules([modulesPaths: ['./path/to/node_modules']])`, `useUnimodules([modulesPaths: ['./path/to/node_modules']])`
```